test/backend_unary_elementwise.in.cpp

```diff
@@ -211,6 +211,27 @@ NGRAPH_TEST(${BACKEND_NAME}, cosh)
     EXPECT_TRUE(test::all_close_f(input, read_vector<float>(result)));
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, erf)
+{
+    Shape shape{8};
+    auto A = make_shared<op::Parameter>(element::f32, shape);
+    auto f = make_shared<Function>(make_shared<op::Erf>(A), ParameterVector{A});
+
+    // TODO: Fix when backends are ready
+    auto backend = runtime::Backend::create("INTERPRETER");
```

Comment:
Instead of hard-coding to `INTERPRETER` we should put `${BACKEND_NAME}` and disable the test in the other backends' manifests.